### PR TITLE
Support multiple events in sandbox

### PR DESCRIPTION
**Pull request by deniskulicek**

### CODE REVIEW

**Comment by joshsmith** [comment]
Can you share some context as to what this change does and what the motivation is?

**Comment by deniskulicek** [comment]
@joshsmith basically this expands the sandbox to store more than one event per event type. 
The way sandbox worked before was as a key-value store where the value was being replaced with the latest event. This PR changes this by simply storing events in lists (under corresponding keys).

https://github.com/joshsmith/segment_elixir/pull/9/files

File: lib/segment/sandbox.ex [hunk 1]
```diff
@@ -3,20 +3,35 @@ defmodule Segment.Sandbox do
...
 
   def track(t = %Track{}) do
-    Agent.update(__MODULE__, &Map.put(&1, "track", t))
+    Agent.update(__MODULE__, fn state -> %{state | track_calls: [t | state.track_calls]} end)
   end
 
-  def get_track do
-    Agent.get(__MODULE__, &Map.get(&1, "track"))
+  def get_track_calls do
+    Agent.get(__MODULE__, &Map.get(&1, :track_calls))
```
Comment (deniskulicek) [inline lib/segment/sandbox.ex:34]: The list that is being returned should be reversed if we care about keeping the order of the events in which they arrived

Comment (joshsmith) [reply lib/segment/sandbox.ex:34]: Should we return it that way ourselves or at least add some documentation to that effect?

Comment (joshsmith) [reply lib/segment/sandbox.ex:34]: @deniskulicek ping on this again

Comment (deniskulicek) [reply lib/segment/sandbox.ex:34]: I can see use-cases that might require order to be preserved, though to keep it simple I'm fine with just making it clear that returned events aren't in order they arrived in.

They can be reversed quite easily when needed
`get_track_calls() |> Enum.reverse()`

**Review by joshsmith — APPROVED** [review]

**Pull request merged**